### PR TITLE
Responsive stuff

### PR DESCRIPTION
**Pull request by andrewhayward**
Tidying up general bits and pieces to work at smaller sizes

### CODE REVIEW

File: gameon/base/static/base/css/home.css [hunk 1]
```diff
@@ -40,6 +40,7 @@
     filter: progid:DXImageTransform.Microsoft.gradient( startColorstr='#000a14', endColorstr='#00283a',GradientType=1 ); /* IE6-9 */
     color: #fff;
 }
+div[role=main] .big-picture .busta,
 .big-picture .busta {
```
Comment (rossbruniges) [inline gameon/base/static/base/css/home.css]: Duplicate selector?

File: gameon/base/static/base/css/home.css [hunk 2]
```diff
@@ -164,6 +165,7 @@
 }
 
 @media all and (max-width: 800px) {
+    div[role=main] .big-picture .busta,
     .big-picture .busta {
```
Comment (rossbruniges) [inline gameon/base/static/base/css/home.css]: Duplicate selector?

File: gameon/base/static/base/css/master.css [hunk 3]
```diff
@@ -383,9 +389,86 @@ nav li a {
...
+    }
+    .shout {
+        font-size: 36px;
+    }
+    .main-col {
+        float: none;
+        width: auto;
+        margin-right: 0;
+    }
+    .mini-col {
```
Comment (rossbruniges) [inline gameon/base/static/base/css/master.css:466]: Might be cool to have some kind of visual divider between the two 'columns' to differentiate the two now they're not side by side and different widths...

File: gameon/base/static/base/css/home.css (hunk 1 above)
Comment (andrewhayward) [inline gameon/base/static/base/css/home.css]: Yes, they kind of are. There was some thought process at the time, I'm sure.

File: gameon/base/static/base/css/home.css (hunk 2 above)
Comment (andrewhayward) [inline gameon/base/static/base/css/home.css]: As above.

File: gameon/base/static/base/css/master.css (hunk 3 above)
Comment (andrewhayward) [inline gameon/base/static/base/css/master.css:466]: OK. Will do.

**Comment by andrewhayward** [comment]
Added a few more commits, to cover your suggestions.

**Comment by rossbruniges** [comment]
Let's see how this looks!

**Pull request merged**